### ISSUE TITLE
Profanity can't list MUC participants without affiliation

### ISSUE THREAD

**Issue by mdosch** [profanity-im/profanity#1425]
If I want to list MUC participants without affiliation profanity doesn't list them.

## Expected Behavior
`/affiliation list none` should list all people in the room without affiliation like `/affiliation list member` lists all people with the affiliation member.

## Current Behavior
`Cannot list users with no affiliation.`

## Steps to Reproduce (for bugs)

1. Perform `/affiliation list none` in a MUC were some participants have no affiliation

## Environment
Debian Bullseye (Testing)

```
Profanity, version 0.9.5dev.master.3d082bfb
Copyright (C) 2012 - 2019 James Booth <boothj5web@gmail.com>.
Copyright (C) 2019 - 2020 Michael Vetter <jubalh@iodoru.org>.
License GPLv3+: GNU GPL version 3 or later <https://www.gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Build information:
XMPP library: libstrophe
Desktop notification support: Enabled
OTR support: Enabled (libotr 4.1.1)
PGP support: Enabled (libgpgme 1.14.0-unknown)
OMEMO support: Enabled
C plugins: Enabled
Python plugins: Enabled (3.8.5)
GTK icons: Enabled
```

**Comment by weiss** [comment]
> `/affiliation list none` should list all people in the room without affiliation like `/affiliation list member` lists all people with the affiliation member.

As affiliations (unlike roles) are unrelated to the list of current participants, `/affiliation list member` will list all members regardless of whether they're currently joined.  Therefore, listing the current participants that aren't on any affiliation list would _not_ be analogous to listing the JIDs with a given affiliation.  Strictly speaking, I'd expect `/affiliation list none` to list all existing JIDs in the world except those who are affiliated with the room :smile:

In other words, I'm not sure the `/affiliation` command is the right place for listing unaffiliated JIDs.  (What's the use case, BTW?)

**Comment by mdosch** [comment]
Some people are particular about getting member status if they are 
regulars in a MUC (don't ask me why, my MUCs are not moderated so it 
doesn't make a difference). With `/affiliation list member` I can see 
which participants have member status, but I didn't find any command 
which lists me the participants who have affiliation `none`.
But to please those people I need to be able to see who does not have 
affiliation `member`. I can go through one by one using conversations 
but it would be a lot more comfortable if I would be able to do this 
using profanity.

How does mcabber do this?

**Comment by weiss** [comment]
Won't a command that simply lists all current participants with some flag that indicates their affiliation, (optionally?) sorted by that flag, do the trick?  (Various other clients have that.)

**Comment by mdosch** [comment]
Am 12. Dezember 2020 09:28:03 UTC schrieb "Holger Weiß" <notifications@github.com>:
>Won't a command that simply lists all current participants with some flag that indicates their affiliation, (optionally?) sorted by that flag, do the trick?  (Various other clients have that.)
>

That would also do it. All I need is to see which participants have no affiliation yet.

**Comment by mdosch** [comment]
Another use case: Sometimes you want to list the participants which have no affiliation yet to give member status to regulars so that they have voice once you have to switch to moderated due to a spam wave.

**Comment by MarcoPolo-PasTonMolo** [comment]
We could insead of `/affiliation list none` have `/who none`, the code for it is pretty much ready
